src/data/text/item_descriptions.h

```diff
@@ -3817,3 +3817,18 @@ static const u8 sLoadedDiceDesc[] = _(
     "Rolls high numbers.\n"
     "Multihit strikes\n"
     "hit more times.");
+
+static const u8 sBlackAuguriteDesc[] = _(
+    "A glassy black\n"
+    "stone that makes\n"
+    "certain Pokémon evolve.");
```

Comment:
Have you checked the descriptions InGame? I highly doubt that 3rd line of the Black Augurite's description looks proper.

Reply:
This is the Ingame description,
> A glassy black stone that produces a sharp cutting edge when split. It’s loved by a certain Pokémon.

I couldn't fit it all in there, maybe do the same as others? 

> Makes certain species of Pokémon evolve.

Reply:
Yeah, the default interface doesn't really leave room for flavor text most of the time, so you'll have to get basic.

Here's an example that works: 

```suggestion
static const u8 sBlackAuguriteDesc[] = _(
    "A black stone that\n"
    "makes some Pokémon\n"
    "evolve.");
```

Reply:
How's this?

```
static const u8 sBlackAuguriteDesc[] = _(
    "A black stone that\n"
    "makes some Pokémon\n"
    "evolve.");;

static const u8 sLinkingCordDesc[] = _(
    "A mysterious string\n"
    "that makes some Pokémon\n"
    "evolve.");

static const u8 sPeatBlockDesc[] = _(
    "A block of material\n"
    "that makes some\n"
    "Pokémon evolve.");

```

Reply:
![image](https://user-images.githubusercontent.com/2904965/221429479-0b69ae6e-c1bf-4958-8296-ceff274a8fa5.png)

Reply:
Moving "Pokémon" to the third line for the Linking Cord will fit
![image](https://user-images.githubusercontent.com/2904965/221429826-6b606462-b44e-413d-98f8-a8c6c9233db3.png)

Reply:
I did edit that, do they sound good? 
also for Black Augurite's item name I removed the second "u" cause, there wasn't enough space. Should I change ITEM_NAME_LENGTH to 16 to support that?

Reply:
done!